src/GoTrueClient.ts

```diff
@@ -352,14 +352,8 @@ export default class GoTrueClient {
       if (error) {
         return { session: null, error: error }
       }
-      if (!data) {
-        return {
-          session: null,
-          error: { name: 'Invalid refresh_token', message: 'JWT token provided is Invalid' },
-        }
-      }
```

Comment:
This turned out to be incompatible with the new signature. However, I don't quite see how this line would even be reached. If the JWT token is invalid then an error should be returned so this function would exit within the `if (error)` branch.

Reply:
@Iamshankhadeep any input on removing this error on `!data` ?

Reply:
@awalias we can remove `!data` as @Andarist said if the token is invalid then this if condition can never be reached. So I am good with omitting `if` condition